tests/src/test/scala/whisk/core/cli/test/SequenceMigrationTests.scala

```diff
@@ -55,6 +54,9 @@ class SequenceMigrationTests
     val wsk = new Wsk
     val whiskConfig = new WhiskConfig(WhiskEntityStore.requiredProperties)
     // handle on the entity datastore
+
+    SharedModules.initSharedModules(List(new SharedModule(actorSystem, whiskConfig, logging)))
```

Comment:
perhaps this should go into a helper trait that gets weaved in with many of tests?  perhaps `WskActorSystem`?